pkgs/top-level/all-packages.nix

```diff
@@ -30910,8 +30910,6 @@ with pkgs;
 
   svox = callPackage ../applications/audio/svox { };
 
-  g4music = callPackage ../applications/audio/g4music { };
```

Comment:
This needs an alias in `top-level/aliases.nix`

Reply:
I made it throw an error, since people should be using the new name now. Hopefully that's ok, if not I can change it to just put a warning, or even make it a normal alias without warning.